tests/phpunit/integration/api/fields/beansRegisterFields.php

```diff
@@ -39,7 +39,7 @@ public function test_should_register_fields() {
 		$this->assertTrue( beans_register_fields( $test_data['fields'], 'beans_tests', $test_data['section'] ) );
 
 		// Check what was registered.
-		$registered = $this->get_reflective_property_value( 'registered' );
+		$registered = $this->get_reflective_property_value( 'registered', '_Beans_Fields' );
```

Comment:
You'll see my trail of notes concerning this one test.  Right now, this is the only test that uses this method. 
 Instead of building a method in the Test Case for this one usage, let's do this instead:

```
$registered_property = $this->get_reflective_property( 'registered', '_Beans_Fields' );
$registered = return $registered_property->getValue( new _Beans_Fields() );
```

Reply:
@christophherr Go ahead and fix it in the WP Customize API as part of this PR.  Then we can group all these changes together.  It'll also get rid of the merge conflict too.